README.md

```diff
@@ -5,7 +5,7 @@
   <img src="mathmagician.jpeg" alt="logo" width="140"  height="auto" />
   <br/>
 
-  <h3><b>Book Store</b></h3>
+  <h3><b>Book Store using React</b></h3>
```

Comment:
- [ ] Please 🙏🏽 remove the Microsoft logo or replace it with your own logo for a professional README file.

Reply:
Done👌